README.md

```diff
@@ -201,6 +203,18 @@ Usage:
 npm run cdk deploy OpenSearch-CI-Dev -- -c useSsl=false -c runWithOidc=false -c additionalCommands='./example.txt'
 ```
 
+#### Use Production Agents
+Please note that if you have decided to use the provided production jenkins agents then please make sure that you are 
+deploying the stack in US-EAST-1 region as the AMIs used are only publicly available in US-EAST-1 region.
+If you want to deploy the stack in another region then please make sure you copy the public AMIs used
+from us-east-1 region to your region of choice and update the code in agent-nodes.ts file accordingly.
```

Comment:
Maybe it should be: `update the new `ami-id` in agent-nodes.ts file`

Reply:
corrected.